dub.json

```diff
@@ -19,7 +19,7 @@
             "targetName": "dstep",
             "sourcePaths": ["dstep", "clang"],
             "importPaths": ["dstep", "clang"],
-            "lflags-posix": ["-lclang", "-rpath", ".", "-L."],
+            "lflags-posix": ["-lclang","-lz", "-rpath", ".", "-L."],
```

Comment:
That's weird. Can you see  in the linker error which function is using libz? Could you please try with DMD as well. I've never compiled DStep with LDC.